### ISSUE TITLE
FE: Fix theme issues on two components

### ISSUE THREAD

**Issue by tomimarkus991** [taltech-vanemarendajaks/team-25-borsibaar#35]
# Theme
Some UI elements contrast ratios fail or don't match theme. These need to be checked and fixed by checking them on https://webaim.org/resources/contrastchecker/

1.  <img width="107" height="58" alt="Image" src="https://github.com/user-attachments/assets/d8243719-e6e7-4f3b-a98e-74af1c23251b" /> Something with yellow background and yellow text would suit better

1. <img width="142" height="54" alt="Image" src="https://github.com/user-attachments/assets/afa3d59c-59ed-4b4b-b531-1077de845b77" />
